Pages/styles.css

```diff
@@ -1,244 +1,117 @@
     display: block;
-       
-    }
+  }
+  .secThreepara:first-child {
+    margin: 0;
+  }
 }
-.swiper {
-    width: 600px;
-    height: 300px;
-    margin-bottom: 80px;
-    bottom: 55px;
-    border-radius: 10px;
+
+h1,
+h2 {
+  color: beige;
+  text-align: center;
+}
+h1 {
+  font-size: 30px;
+}
+h2 {
+  font-size: 20px;
+  color: beige;
 }
-.swiper-slide img {
-    height: 400px;
-    width: inherit;
-}
- .secTwoimages ul {
-     display: flex;
-     flex-grow: 1;
-     justify-content: space-around;
-     padding: 0;
-     
- }    
- ul li {
-     list-style-type: none;
-     display: inline-flex;
-     padding: 5px;
- }
- .img-1, .img-2, .img-3,
- .img-4, .img-5, .img-6 {
-     display: none;
-    
- }
- @media (max-width: 680px) {
-    .img-1, .img-2, .img-3,
-    .img-4, .img-5, .img-6 {
-        height: 100px;
-        width: 90px;
-        border-radius: 20px;
- }
- }
 
- .articleImage {
-     float: left;
-     border-radius: 10px;
-     padding: 25px;
-     height: 200px;
- }
- @media (max-width: 680px) {
-    .articleImage {
-        float: none;
-        padding-left: 30px;
-        padding-top: 20px;
-    }
- }
-@media (max-width: 680px) {
-    .articleImage2 {
-        float: none;
-        padding-left: 30px;
-        padding-top: 20px;
-    }
+nav {
+  text-align: center;
+}
+nav ul {
+  padding: 0;
+  margin: 4px;
+  font-size: 20px;
+  color: bisque;
+}
+nav ul li {
+  font-size: 14px;
+  list-style-type: none;
+  display: inline-flex;
+  padding: 5px;
 }
 
- .articleImage2 {
-     height: 200px;
-     float: left;
-     padding: 25px;
-     
- }
- secThreepara {
-    width: 50%;
-}
- .secThreepara p {
-     font-size: 20px;
-     columns: 2;
-     color: rgb(188, 188, 130);
-     font-family: 'Playfair Display', serif;
- }
- 
+section {
+  margin-bottom: 48px;
+}
 
+.profileImage {
+  height: 100%;
+  width: auto;
+  border-radius: 10px;
+}
 
-/*.img-1, .img-2, .img-3,
-.img-4, .img-5, .img-6 {
-    height: 100px;
-    width: 95px;
-    border-radius: 20px;
-}
-.articleImage {
-    float: left;
-    border-radius: 10px;
-    padding: 25px;
-    height: 200px;
-}
-.articleImage2 {
-    height: 200px;
-    float: left;
-    padding: 25px;
-    
-}
-.secTwopara p {
-    height: 125px;
-    font-size: 14px;
-    justify-content: ;
+.secTwo {
+  height: 250px;
+  width: 600px;
+  margin: auto;
+}
+.secTwopara {
+  font-family: "Playfair Display", serif;
+  font-size: 12px;
+  height: 100%;
+  margin-left: 14px;
+  /* width: 300px; */
+  background-color: #e7d0b3;
+  border-radius: 10px;
+  padding: 12px;
+}
+.secTwoScroll {
+  height: 100%;
+  overflow: auto;
 }
 
-    .secTwopara p { 
-    margin: 0 2px 8px;
-    height: 125px;
-    font-size: 12px;
-    text-indent: 15px;
-    
-}
\ No newline at end of file
+.swiper {
+  width: 600px;
+  height: 300px;
+}
+.swiper-slide img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.secThreepara {
+  width: 50%;
+}
+.secThreepara:first-child {
+  margin-right: 50px;
```

Comment:
add a margin between the first and second column. note that some text seems to flow outside the edge of the box. you can demo this by setting a `background-color` on `.secThreepara`